### PR TITLE
space-age: add exercise with test generator

### PR DESCRIPTION
**Pull request by ferhatelmas**

### CODE REVIEW

File: exercises/space-age/.meta/gen.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"log"
+	"text/template"
+
+	"../../../gen"
+	space "github.com/exercism/go/exercises/space-age"
+)
```
Comment (leenipper) [inline exercises/space-age/.meta/gen.go]: Recommend this change:
```
-       space "github.com/exercism/go/exercises/space-age"
+       space "../../space-age"
```
I think it best to keep the path to the dependent package relative, just as the path to the 'gen' package is relative.

Comment (ferhatelmas) [reply exercises/space-age/.meta/gen.go]: Done.

File: config.json [hunk 2]
```diff
@@ -1151,7 +1163,6 @@
     }
   ],
   "foregone": [
-    "space-age",
```
Comment (ferhatelmas) [inline config.json]: Why was it foregone ?

Comment (tleen) [reply config.json]: Unknown, looks like it was a while ago. @soniakeys do you remember why it was ignored in 92cfaeb99ae95e12ab0c1fbc6f673321aad4b1a0?

**Pull request merged**